storage/badger/badger.go

```diff
@@ -0,0 +1,76 @@
+package badger
+
+import (
+	v3Badger "github.com/dgraph-io/badger/v3"
+	"github.com/pastelnetwork/go-commons/errors"
+	"github.com/pastelnetwork/walletnode/storage"
+)
+
+// badgerDB structure represents wrapper over the badger.badgerDB
+type badgerDB struct {
+	db     *v3Badger.DB
+	logger *Logger
+	config *Config
+}
+
+// NewBadgerDB creates an instance of badgerDB
+func NewBadgerDB(cfg *Config) storage.KeyValue {
+	return &badgerDB{
+		db:     &v3Badger.DB{},
+		logger: NewLogger(),
+		config: cfg,
+	}
+}
+
+// Init method opens the v3Badger db instance using specified configuration data and logger instance
+func (db *badgerDB) Init() (err error) {
+	db.db, err = v3Badger.Open(v3Badger.DefaultOptions(db.config.Dir).WithLogger(db.logger))
+	if err != nil {
+		return errors.Errorf("can't fetch database from %v | %v", db.config.Dir, err)
```

Comment:
```suggestion
		return errors.WithField("db-dir", db.config.Dir).Errorf("could not open database, %v", err)
```

Error messages should be written in the past tense. 
But git-commit messages in the present tense, [read here](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages#:~:text=7%20Answers&text=The%20preference%20for%20present%2Dtense,messages%20comes%20from%20Git%20itself.&text=Describe%20your%20changes%20in%20imperative,codebase%20to%20change%20its%20behavior.)

And you have everything the other way around :) please fix all error messages